docs/approach.md

```diff
@@ -0,0 +1,22 @@
+# Ring Vaccination
+
+Ring vaccination is a public health strategy to control the spread of an infectious disease by vaccinating individuals who have been exposed to the disease. In a ring vaccination strategy, after a case has been confirmed, health officials try to identify the patient's close contacts and vaccinate them. Ring vaccination also includes identifying and vaccinating the contacts of those contacts to form a protective "ring" of immunity around infected individuals.  This strategy relies on a few key factors:
+
+- quickly find contacts before they become infectious, i.e. short delays between case identification and diagnosis
+- contact tracing a high fraction of close contacts
+- effective isolation of diagnosed cases
+- quick vaccination of contacts
+- contact tracing and vaccination of a high fraction of contacts of contacts
+- assumes that post exposure vaccination is effective at preventing transmission
```

Comment:
@pearsonca do you think that it's sufficient for us to briefly mention these points as caveats to the effectiveness of ring vaccination? I think the purpose of this is to give enough detail to a fellow modeler so they can use the widget, but not have to go into all model details. I guess what I'm asking is: would you like for me to find a way to incorporate your points into this lit review/approach doc?

Reply:
I think it may be useful context for your audience, but you're going to know them better than I.

If your inclined to address these kind of issues, I'd say ... think through your model assumptions carefully (I have not done that *for your model*, just highlighting what I've seen over and over for these kind of models + the implication of afmcgee42's Q).

If those assumptions imply that, logically, ring vaccination isn't going to get credit for things that are in the real world but are precluded in your model world, then it may be reasonable to indicate to your audience that the estimated benefits are underestimated & then explain why.

Reply:
I think @pearsonca 's are good points to add, that this simulation is only looking at *transmission*. Ring vaccination might be a good idea to prevent *disease*, even if it did nothing to control transmission.

Reply:
To be clear: I think there are likely *some* transmission-preventing benefits to ring vaccination _in the real world_, but that typical basic model worlds are framed such that you can logically conclude, within those worlds, there isn't going to be *any* benefit.

(at the same time, I think the people that are ra-ra-ring-vaccine in terms of controlling outbreaks aren't dealing enough with the cold logic of models which are not _bad_ approximations of the real world: there's probably a lot less benefit to control than they imagine)

Reply:
Putting it a bit concretely as well: in some sense, vaccinating contacts-of-contacts is a way to ensure the assumed efficacy of quarantine of contacts. They might not be fully quarantine compliant, but their likeliest contacts have additional protection against exposure.